docs/DeveloperGuide.adoc

```diff
@@ -806,9 +806,59 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |sort contacts based on their locality |better match students to tutors
+
+|`* * *` |user |edit the information stored |keep the information up to date
+
+|`* * *` |user |tag tutors and students with multiple values |
+
+|`* * *` |user |add contacts of both tutors and students |
+
+|`* * *` |user |categorise contacts between students and tutors |
+
+|`* * *` |user |group clients based on the subject they wish to study/teach |
+
+|`* * *` |user |record what preference the students/tutors have (male/female) |
+
+|`* * *` |user |see and compare the price that students/tutors are willing to pay/accept|match them based on pricing
+
+|`* * *` |user |have an autosave function |changes will be saved once i exit the application
```

Comment:
Replace the `i` with `the user`. User stories should be about the user.

Reply:
updated